dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/JacksonEmptyContainerLoader.java

```diff
@@ -84,9 +85,22 @@ private Optional<Object> constructEmptyInstance(Type type, TypeMarker<?> origina
             }
         }
 
+        Constructor<?> emptyRecordConstructor = getEmptyRecordCanonicalConstructor(type);
```

Comment:
Not exactly related to this PR, but we'll need to remember to register these with reachability-metadata!